standalone/ansible/roles/sidekiq/tasks/main.yml

```diff
@@ -21,3 +21,4 @@
   shell: monit restart "{{ sidekiq_service_name }}"
   tags: restart-sidekiq
   become: true
+  ignore_errors: yes
```

Comment:
Why ignore errors?

Reply:
monit takes a while to load up sidekiq (upto 5 minutes, at times) and immediately issuing this restart fails because it doesn't know about the process yet. This happens only on the first setup run. The restart works well on subsequent deploys.

There might be a cleaner way to do this but felt this didn't warrant the effort.

Reply:
Ignoring errors seems risky. Are the errors at least logged so the dev knows something went wrong? I'm wondering if we'll ever hit a situation where we want to restart Sidekiq to pick up some env var change or something, and don't notice that it didn't work.

This is only an issue if we choose to restart Sidekiq within a few minutes of starting it up, right? Could we live with that instead?

Reply:
The errors are logged, yes. This task needs to be here, since we dont' have `bundle exec sidekiq:restart` anymore. 

The other options are:
- Add a 5 minute delay,
- Using `async`: not great because the errors won't be logged if it fails.
- We might be able to get away with some refactoring (split the sidekiq tasks between `deploy` role and `setup` role) but that'll be a slighly larger change.

Reply:
If the errors are logged, then this is fine 👍 🚀